geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStore.scala

```diff
@@ -83,7 +83,10 @@ class KafkaDataStore(val config: KafkaDataStoreConfig)
           } else {
             config.indices.cqAttributes
           }
+          // note: handles points vs non-points internally
           GeoCQIndexSupport(sft, attributes, config.indices.resolutionX, config.indices.resolutionY)
+        } else if (sft.nonPoints) {
+          SizeSeparatedBucketIndexSupport(sft, config.indices.resolutionX / 360d, config.indices.resolutionY / 180d)
```

Comment:
Looks like the SizeSeparatedBucket is not compatible with the CQEngine support?

Reply:
it is, but it gets handled by cqEngine (see note two lines up)